apps/api/src/persistence/database/database.provider.ts

```diff
@@ -33,6 +33,22 @@ export class ArangoDatabaseProvider {
         return new ArangoDatabaseForCollection<TEntity>(this.arangoInstance, collectionName);
     };
 
+    /**
+     * This is a test helper. It will not throw if you attempt to call it outside
+     * of a testing environment.
+     */
+    clearViews = async (): Promise<void> => {
```

Comment:
Do we have a test for this?

Reply:
Note that this is not as destructive as it seems. We can always replay the events to rehydrate the state. That said, I have added a test.